examples/pyth/programs/pyth/src/pc.rs

```diff
@@ -0,0 +1,103 @@
+use std::cell::RefMut;
+
+use crate::*;
+use anchor_lang::prelude::AccountInfo;
+use bytemuck::{cast_slice_mut, from_bytes_mut, try_cast_slice_mut, Pod, Zeroable};
+
+#[derive(Default, Copy, Clone)]
+#[repr(C)]
+pub struct AccKey {
+    pub val: [u8; 32],
+}
+
+#[derive(Copy, Clone)]
+#[repr(C)]
+pub enum PriceStatus {
+    Unknown,
+    Trading,
+    Halted,
+    Auction,
+}
+impl Default for PriceStatus {
+    fn default() -> Self {
+        PriceStatus::Trading
+    }
+}
+
+#[derive(Copy, Clone)]
+#[repr(C)]
+pub enum CorpAction {
+    NoCorpAct,
+}
+impl Default for CorpAction {
+    fn default() -> Self {
+        CorpAction::NoCorpAct
+    }
+}
+#[derive(Default, Copy, Clone)]
+#[repr(C)]
+pub struct PriceInfo {
+    pub price: i64,
+    pub conf: u64,
+    pub status: PriceStatus,
+    pub corp_act: CorpAction,
+    pub pub_slot: u64,
+}
+#[derive(Default, Copy, Clone)]
+#[repr(C)]
+pub struct PriceComp {
+    publisher: AccKey,
+    agg: PriceInfo,
+    latest: PriceInfo,
+}
+
+#[derive(Copy, Clone)]
+#[repr(C)]
+pub enum PriceType {
+    Unknown,
+    Price,
+    TWAP,
+    Volatility,
+}
+impl Default for PriceType {
+    fn default() -> Self {
+        PriceType::Price
+    }
+}
+#[derive(Default, Copy, Clone)]
+#[repr(C)]
+pub struct Price {
+    pub magic: u32,       // pyth magic number
```

Comment:
Please start and end all comments with `Uppercase and a period.`

Reply:
Thanks will do.